src/components/base/text-area/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import propTypes from "prop-types";
+
+function TextArea({ noOfRows, register, error }) {
+  return (
+    <div>
+      <textarea
+        name="data"
+        {...register}
+        className="width-100"
+        rows={noOfRows}
```

Comment:
ditto.
FYI: Ditto is used to indicating that something has already been said or in this case reviewed earlier.